ui/client/components/graph/node-modal/editors/expression/Editor.ts

```diff
@@ -22,7 +22,7 @@ import TextareaEditor from "./TextareaEditor"
 import JsonEditor from "./JsonEditor"
 import DualParameterEditor from "./DualParameterEditor"
 
-type ParamType = $TodoType
+export type ParamType = $TodoType
```

Comment:
Maybe let's move it to type definitions if we want to export it?

Reply:
👍

Reply:
@mproch 
https://github.com/TouK/nussknacker/pull/1191/commits/6f6b606889c4cc90a05b21c6b8c8012a468b061e